mobile_verifier/src/service_provider/promotions.rs

```diff
@@ -0,0 +1,51 @@
+use mobile_config::client::{carrier_service_client::CarrierServiceVerifier, ClientError};
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+
+use crate::service_provider::ServiceProviderId;
+
+mod proto {
+    pub use helium_proto::{service_provider_promotions::Promotion, ServiceProviderPromotions};
+}
+
+pub async fn get_promotions(
+    client: &impl CarrierServiceVerifier<Error = ClientError>,
+) -> anyhow::Result<ServiceProviderPromotions> {
+    let promos = client.list_incentive_promotions().await?;
+    Ok(ServiceProviderPromotions(promos))
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct ServiceProviderPromotions(Vec<proto::ServiceProviderPromotions>);
```

Comment:
i kinda like this wrapper and also dont

Reply:
Agreed. The plural works at both levels. I was hoping someone might help with a better name for the proto, and the wrapper name would become clearer.